bundle/compliance/cis_aws/rules/cis_3_11/test.rego

```diff
@@ -0,0 +1,43 @@
+package compliance.cis_aws.rules.cis_3_11
+
+import data.cis_aws.test_data
+import data.compliance.cis_aws.data_adapter
+import data.lib.test
+
+s3_object_type = "AWS::S3::Object"
+
+not_s3_object_type = "AWS::S3ObjectLambda::AccessPoint"
+
+test_violation {
+	eval_fail with input as rule_input(null, true)
+	eval_fail with input as rule_input([], true)
+	eval_fail with input as rule_input([{"ReadWriteType": "WriteOnly"}], true)
+	eval_fail with input as rule_input([{"ReadWriteType": "WriteOnly", "DataResources": [{"Type": not_s3_object_type}]}], true)
+	eval_fail with input as rule_input([{"ReadWriteType": "ReadOnly", "DataResources": [{"Type": not_s3_object_type}]}], true)
+	eval_fail with input as rule_input([{"ReadWriteType": "WriteOnly", "DataResources": [{"Type": s3_object_type, "Values": ["arn:aws:s3"]}]}], true)
+	eval_fail with input as rule_input([{"ReadWriteType": "ReadOnly", "DataResources": [{"Type": s3_object_type, "Values": ["arn:aws:s3"]}]}], false)
+}
+
+test_pass {
```

Comment:
in the remediation section they mention other fields, are they relevant for the rule evaluation too?
```
aws cloudtrail put-event-selectors --region <region-name> --trail-name <trail-name> --event-selectors '[{ "ReadWriteType": "ReadOnly", "IncludeManagementEvents":true, "DataResources": [{ "Type": "AWS::S3::Object", "Values": ["arn:aws:s3:::<s3-bucket-name>/"] }] }]'
```

Reply:
I don't think we should address `IncludeManagementEvents ` as part of the rule, it's enabled by default ([docs](https://docs.aws.amazon.com/awscloudtrail/latest/APIReference/API_EventSelector.html))
Furthermore, In rules 3.10 & 3.11 it was not mentioned in the audit section.